pyocd/probe/stlink/stlink.py

```diff
@@ -131,7 +131,10 @@ def get_version(self):
             #   8-9: ST_VID
             #   10-11: STLINK_PID
             response = self._device.transfer([Commands.GET_VERSION_EXT], readSize=12)
-            hw_vers, _, self._jtag_version = struct.unpack('<3B', response[0:3])
+            hw_vers, _, self._jtag_version, self._msc_version = struct.unpack('<4B', response[0:4])
+
+        self._version_str = "V%dJ%dM%d" % (self._hw_version, self._jtag_version, self._msc_version)
+        LOG.debug("STLink probe %s firmware version: %s", self.serial_number, self._version_str)
```

Comment:
Seems that "LOG" is not compiling :-)

and maybe info level could be OK ?

Reply:
Oops! I cherry picked this change from another branch where I've cleaned up the log usage. I'll just wait until that branch is merged in to complete this PR.

The reason I used debug level is that I try to keep info level output to the minimum. If the STLink version is old enough that it won't work, a warning will be logged. Otherwise the user probably doesn't need to know the version.